src/Raven.Server/RavenServer.cs

```diff
@@ -1010,18 +1011,18 @@ public bool RefreshClusterCertificate(object state, string raftRequestId)
             var forceRenew = state as bool? ?? false;
 
             var currentRefreshTask = _currentRefreshTask;
-            if (currentRefreshTask.IsCompleted == false)
+            if (currentRefreshTask.IsValueCreated == false || currentRefreshTask.Value.IsCompleted == false)
```

Comment:
`currentRefreshTask.IsValueCreated == false` will always return `true`, right?
It's not a big deal, but it can be a bit confusing when reading the code and trying to figure out when it could be `false`.